.github/workflows/test.yaml

```diff
@@ -0,0 +1,80 @@
+name: build
+
+on:
+  push:
+    branches: [ "*" ]
+  pull_request:
+    branches: [ master ]
```

Comment:
We used `*` for all branches of pushes and PRs in every other repo, so why are we now putting `master` on each of these PRs? @LucasRoesler do you know why?

Reply:
done, thank you 🙏